smoke-test/tests/cli/datahub-cli.py

```diff
@@ -57,10 +57,10 @@ def test_rollback_editable():
     session.post(rollback_url, data=json.dumps({"runId": ingested_dataset_run_id, "dryRun": False, "hardDelete": False}))
 
     # Allow async MCP processor to handle ingestions & rollbacks
-    sleep(10)
+    sleep(20)
```

Comment:
We need to sleep 20 seconds?

Reply:
10 seconds has lead to some flakiness, in our CI so I thought to increase it for stability purposes.

Reply:
holy